test/app.spec.js

```diff
@@ -123,6 +123,25 @@ describe('app', () => {
     });
   });
 
+  describe('start', () => {
+    beforeEach(() => {
+      sinon.stub(messenger.app, 'listen');
```

Comment:
🚲  ⚠️ 

This is only one stub so it isn't a big deal, but it's pretty common throughout the Condé internal codebases to use `sandbox = sinon.sandbox.create();` here with a variable scoped to the `describe` then use `sandbox.stub()` and so on. It simplifies the `afterEach` as you only have to call `sandbox.restore()` to get everything back to the original state.